gc/scheduler/scheduler.go

```diff
@@ -305,6 +306,13 @@ func (s *gcScheduler) run(ctx context.Context) {
 		}
 
 		s.waiterL.Lock()
+		ctx, gcRunSpan := tracing.StartSpan(ctx, tracing.Name(gcSpanPrefix, "run"))
+		gcRunSpan.SetAttributes(
```

Comment:
This PR #7616  defines `tracing.WithAttribute` option to set attributes to a span when it is created. Once that PR is merged, we should update this as well. I am fine with doing it in a follow up PR.

Reply:
ok,I will modify it after the pr is merged.